packages/status-js/src/client/community/community.test.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * Integration test suite
+ */
+
+// todo?: move to __tests__/, tests/
+
+import { beforeEach, describe, expect, test, vi } from 'vitest'
+
+import { Community } from './community'
+
+import type { Waku } from 'js-waku'
+
+// todo: move to __mocks__
+// todo?: mock all methods with one func; vi.mock('')
+const WakuMock = {
+  store: {
+    queryHistory: vi.fn(),
+  },
+  relay: {
+    addObserver: vi.fn(),
+    addDecryptionKey: vi.fn(),
+    deleteObserver: vi.fn(),
+    send: vi.fn(),
+  },
+} as unknown as Waku
+
+describe('Community', () => {
+  const community = new Community(
+    // fixme!:
+    null!,
+    '0x029f196bbfef4fa6a5eb81dd802133a63498325445ca1af1d154b1bb4542955133'
+  )
+
+  beforeEach(() => {
+    // todo?: set as config instead
+    vi.resetAllMocks()
+  })
+
+  // todo?: use `it`
+  test('is owner', () => {
```

Comment:
Note: still not covered according to the report.

Reply:
See:

- https://github.com/istanbuljs/nyc/issues/711
- https://github.com/istanbuljs/istanbuljs/issues/291
- https://github.com/bcoe/c8/issues/404
- https://github.com/bcoe/c8/issues/322
- https://github.com/facebook/jest/issues/12873
- https://github.com/istanbuljs/istanbuljs/issues/547